packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -187,11 +187,18 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
       onOverlayClickNoCalendar={() => {
         router.push("/apps/categories/calendar");
       }}
-      onClickOverlayContinue={() => {
-        const newUrl = new URL(`${WEBAPP_URL}/login`);
-        newUrl.searchParams.set("callbackUrl", window.location.pathname);
-        newUrl.searchParams.set("overlayCalendar", "true");
-        router.push(newUrl.toString());
+      onClickOverlayContinue={(provider: "calcom" | "google" = "calcom") => {
+        const currentUrl = new URL(`${WEBAPP_URL}/login`);
+        currentUrl.searchParams.set("overlayCalendar", "true");
+        if (provider === "google") {
+          const url = new URL("/getting-started/connected-calendar", window.location.origin);
+          url.searchParams.set("callbackUrl", currentUrl.toString());
+          signIn("google", { callbackUrl: url.toString() });
+        } else {
+          currentUrl.pathname = "/login/";
+          currentUrl.searchParams.set("callbackUrl", window.location.pathname);
+          router.push(currentUrl.toString());
+        }
```

Comment:
Tagging @sean-brydon to verify this doesn't conflicts with his recent work here.